docs/integration/jvm-logs.md

```diff
@@ -0,0 +1,33 @@
+title: JVM Logs Integration
```

Comment:
It is just "JVM Logs" or specifically "JVM Garbage Collection Logs"?

Reply:
Adjusted, thanks for pointing that out.